### DISCUSSION TITLE
hachette mr & madame french

### DISCUSSION THREAD

**Discussion by Hermi16** [oyooyo/audiocube#16] [General]
Hello, I bought the audio cube Mr/Mme in French. I have the smp files if you wanted.

**Comment by oyooyo** [comment]
Hello @Hermi16 , thanks for the offer. If I understand correctly, the SMP files from the Mr/Mme can be successfully encrypted/decrypted using the "hachette_green" setting. If this is correct then I do not need the SMP files - I would only need them if this version uses a new kind of encryption.

**Comment by Hermi16** [comment]
I have the Smp files (on the SD card of the audiocube and on my computer), I don't code too well, I would like to learn how to write on RFID cards (to replace the figurines)

**Comment by Hermi16** [comment]
I already know how to convert smp to mp3

**Comment by thomas-runser** [comment]
@Hermi16 
Hello,
I would like to accept your offer, is it possible to share the files? I am missing half of it with the version I bought.
30 to 60 are missing.
Thank you

**Comment by n-harari** [comment]
Hi! These figurines were sold also in Italy. I have the full collection and the micro sd cards, but my cube audio reader broke down and I am having the hardest time finding a replacement! Any advice? Thank you!

**Comment by bonbonieragr-dotcom** [comment]
@Hermi16  Hello! I am looking for the French Hachette Mr/Mme .smp files to use them on my audio cube. Could you please share them with me via a Google Drive or MEGA link? Thank you very much in advance!

**Comment by bonbonieragr-dotcom** [comment]
@n-harari Hi! Since you have the full Italian collection on your micro SD, would you be able to share a backup link (Google Drive/MEGA) with the files? I can also help you with advice on how to fix or replace your broken cube reader if you want!
